english/tos/tos.md

```diff
@@ -0,0 +1 @@
+[Tree of Savior](https://treeofsavior.com/) is the spiritual successor to the classic MMORPG [Ragnarok Online](iro.ragnarokonline.com/)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.